arangod/Aql/Executor/ModificationExecutorHelpers.h

```diff
@@ -94,7 +94,8 @@ OperationOptions convertOptions(ModificationOptions const& in,
                                 Variable const* outVariableNew,
                                 Variable const* outVariableOld);
 
-AqlValue getDocumentOrNull(velocypack::Slice elm, std::string const& key);
+AqlValue getDocumentOrNull(velocypack::Slice elm, std::string const& key,
+                           ResourceMonitor* rm = nullptr);
```

Comment:
Why is this defaulted to nullptr?
If this stays nullptr we are not tracking those slices.
Please make this non default